repo/packages/E/elasticsearch/0/config.json

```diff
@@ -0,0 +1,95 @@
+{
+  "properties":{
+    "mesos":{
+      "depscription":"Mesos specific configuration properties",
+      "properties":{
+        "master":{
+          "default":"zk://master.mesos:2181/mesos",
+          "description":"The URL of the Mesos master. This should point to the Mesos ZNode on the Zookeeper machine.",
+          "type":"string"
+        }
+      },
+      "required":[
+        "master"
+      ],
+      "type":"object"
+    },
+    "elasticsearch":{
+      "depscription":"Elasticsearch specific configuration properties",
+      "properties":{
+        "framework-name":{
+          "default":"elasticsearch",
+          "description":"The framework name.",
+          "type":"string"
+        },
+        "port":{
+          "default":"8080",
+          "description":"The port number of the ES-Mesos management front-end. Defaults to 8080.",
+          "type":"string"
+        },
+        "executor":{
+          "properties":{
+            "ram":{
+              "default":"2048",
+              "description":"The amount of ram that Mesos shall give give to the executors.",
+              "type":"string"
+            },
+            "instances":{
+              "default":"3",
+              "description":"The number of elasticsearch nodes to be instantiated. It is recommended that you provide an odd number, to prevent split brain issues.",
+              "type":"string"
+            }
+          },
+          "required":[
+            "ram",
+            "instances"
+          ],
+          "type":"object"
+        },
+        "scheduler":{
+          "properties":{
+            "cpu":{
+              "default":"0.2",
+              "description":"The amount of CPU that Mesos shall allocate to the scheduler.",
+              "type":"string"
+            },
+            "ram":{
+              "default":"512",
+              "description":"The amount of RAM that Mesos shall allocate to the scheduler.",
+              "type":"string"
+            },
+            "instances":{
+              "default":"1",
+              "description":"The number of scheduler instances.",
+              "type":"string"
+            },
+            "java-heap":{
+              "default":"-Xms128m -Xmx256m",
+              "description":"The java heap space environmental variables to pass to the scheduler",
+              "type":"string"
+            }
+          },
+          "required":[
+            "cpu",
+            "ram",
+            "instances",
+            "java-heap"
+          ],
+          "type":"object"
+        }
+      },
+      "required":[
+        "framework-name",
+        "port",
+        "executor",
+        "scheduler"
+      ],
+      "type":"object"
+    },
+    "required":[
```

Comment:
this required block is at the wrong location... it is 1 more outer block.

you can use the `./1-validate-packages.sh ` in the scripts directory to confirm the files are correct.